woocommerce/woocommerce-ios
# Order Filters: removed status filters from search view

Part of #5243

## Description
As part of the Order Filters implementation, we want to remove the status cell shown inside the Order Search view.
Now the Order Search View will display the list of orders instead that a list of statuses. Plus, I removed all the code related to the view controller for displaying the filters.

## Testing
1. Launch the app.
2. Tap the order tab.
3. Tap the search button.
4. You should be able to see a list of orders instead of a list of statuses, and you should be able to search between orders like before.

## Screenshots

| Before             |  After |
:-------------------------:|:-------------------------:
![IMG_CCAED566E5EA-1](https://user-images.githubusercontent.com/495617/142001118-3f4d9f23-9162-4174-b39f-42fcd79f7c1c.jpeg) | ![Simulator Screen Shot - iPhone 13 Pro - 2021-11-16 at 15 12 24](https://user-images.githubusercontent.com/495617/142001129-d59bc443-3bd9-4202-a2ba-d0b0e36b0607.png)

Update release notes:
